src/app/core/config/config-fix.ts

```diff
@@ -470,9 +470,10 @@ export const defaultJsonConfig = {
           "id": "name"
         },
         {
-          "view": "DisplayText",
+          "view": "DisplayAge",
           "label": $localize`:Column label for age of child:Age`,
-          "id": "age"
+          "id": "age",
```

Comment:
how about switching this to accept `id: "dateOfBirth"`?

Reply:
The problem is, we cannot have to times the same `"id"`. And `dateOfBirth` is already reserved for displaying the actual date, while this one should show the age. We could also put them both in one display component.

Reply:
This makes the config a bit cumbersome ... but I get the problem and have no idea for an alternative :+1: